### PR TITLE
Fallback to local env vars when AWS credentials are not found

### PR DESCRIPTION
**Pull request by mkubala**
When AWS credentials are not defined, we try to load configuration from the local env using typesafe config.

Some of the AWS-specific clients are now marked as optional, especially for DynamoDB which is responsible for results reporting, although providing some naive reporters, to be used on local machine is out of scope for this PR.

### CODE REVIEW

File: src/main/scala/com/softwaremill/mqperf/Receiver.scala [hunk 1]
```diff
@@ -27,19 +27,18 @@ object Receiver extends App {
   }
 }
 
-class ReceiverRunnable(
-  mq: Mq,
-  reportResults: ReportResults,
-  receiveMsgBatchSize: Int) extends Runnable {
+class ReceiverRunnable(mq: Mq,
```
Comment (kciesielski) [inline src/main/scala/com/softwaremill/mqperf/Receiver.scala:30]: [minor] Previous formatting looked compatible with default scalariform output. I believe it's also standard formatting returned by scalafmt, so maybe we should keep it? (and add scalariform/scalafmt by the way :)

File: src/main/scala/com/softwaremill/mqperf/config/TestConfigOnS3.scala [hunk 2]
```diff
@@ -1,71 +1,86 @@
...
 }
 
 object TestConfigOnS3 {
-  def create(args:Array[String]):TestConfigOnS3 = {
-    if(args.length == 0) {
+
+  val LocalConfig: Config = ConfigFactory.load()
+
+  def create(args: Array[String]): TestConfigOnS3 = {
+    if (args.length == 0) {
```
Comment (kciesielski) [inline src/main/scala/com/softwaremill/mqperf/config/TestConfigOnS3.scala:83]: How about `isEmpty`?

Comment (mkubala) [reply src/main/scala/com/softwaremill/mqperf/config/TestConfigOnS3.scala:83]: 👍

File: src/main/scala/com/softwaremill/mqperf/Receiver.scala (hunk 1 above)
Comment (mkubala) [reply src/main/scala/com/softwaremill/mqperf/Receiver.scala:30]: I'm using default Idea formatter rules, just removed a new line, right after opening bracket.

But I'm open for the idea of using some common rules/tool (let me introduce it in a separate PR)

Comment (kciesielski) [reply src/main/scala/com/softwaremill/mqperf/Receiver.scala:30]: Sure 👍

**Pull request merged**